infection_monkey/exploit/struts2.py

```diff
@@ -9,176 +9,66 @@
 import re
 
 import logging
-from exploit import HostExploiter
-from exploit.tools import get_target_monkey, get_monkey_depth
-from tools import build_monkey_commandline, HTTPTools
-from model import CHECK_LINUX, CHECK_WINDOWS, POWERSHELL_HTTP, WGET_HTTP, EXISTS, ID_STRING, RDP_CMDLINE_HTTP, \
-                  DROPPER_ARG
+from web_rce import WebRCE
+import copy
 
 __author__ = "VakarisZ"
 
 LOG = logging.getLogger(__name__)
 
 DOWNLOAD_TIMEOUT = 300
 
-class Struts2Exploiter(HostExploiter):
+
+class Struts2Exploiter(WebRCE):
     _TARGET_OS_TYPE = ['linux', 'windows']
 
     def __init__(self, host):
         super(Struts2Exploiter, self).__init__(host)
-        self._config = __import__('config').WormConfiguration
-        self.skip_exist = self._config.skip_exploit_if_file_exist
-        self.HTTP = [str(port) for port in self._config.HTTP_PORTS]
 
     def exploit_host(self):
```

Comment:
It seems like this should be the flow of all web-rce exploits, why not have it as the actual implementation of exploit_host on WebRCE?
If there's some data that's missing (like parameters to get_ports_w), maybe add an abstract getter for these parameters.